scheduler/wmap/wmap.go

```diff
@@ -240,6 +292,45 @@ type ProcessWorkflowMapNode struct {
 	Target string                 `json:"target"yaml:"target"`
 }
 
+func (pw *ProcessWorkflowMapNode) UnmarshalJSON(data []byte) error {
+	t := make(map[string]json.RawMessage)
+	if err := json.Unmarshal(data, &t); err != nil {
+		return err
+	}
+	for k, v := range t {
+		switch k {
+		case "plugin_name":
+			if err := json.Unmarshal(v, &pw.Name); err != nil {
```

Comment:
for the built-in type, do you like to make the error message consistent as what you did in core/task.go line 178, such as:
```
return fmt.Errorf("%v (while parsing 'name')", err)
```